### ISSUE TITLE
Notebooks that need revision and updating ...

### ISSUE THREAD

**Issue by mmcky** [QuantEcon/QuantEcon.applications#10]
The following Julia notebooks need revision and updating

  - ``lqcontrol_solutions_jl.ipynb``, 
  - ``lqramsey_solutions_jl,ipynb``, 
  - ``rational_expectations/ree_solutions_jl.ipynb``

**Comment by mmcky** [comment]
@cc7768 got a shortcut for the heatmap working in ``arellano`` notebook. It is a temporary fix, but it works for now.

For some reason it wanted the x and y arguments and wasn't happy with the x_min, x_max, y_min, y_max. Should sort out why later.

**Comment by albep** [comment]
Hey @mmcky, do you remember what kind of revision/updating those notebooks needed? `lqramsey_solutions_jl` looks good to me, maybe the others need Markdown text reformatting?

**Comment by mmcky** [comment]
Hi @albep Does the notebook actually execute without errors? From memory these notebooks didn't run.

**Comment by albep** [comment]
Thanks  @mmcky, I tried running `lqcontrol_solutions_jl.ipynb` and it doesn't run.

@guanlong1989 in PR #36 started fixing the issue. However, as he points out, there is (at least) another bug in line 32, In [3] `income = squeeze(wp[1, 2:end], 1) .+ m1 .* time .+ m2 .* time.^2 `.

I believe this is caused by this line of code https://github.com/QuantEcon/QuantEcon.jl/blob/5d25a26daf9a4e7858b7b29d8dfb505e7f0134fc/src/lqcontrol.jl#L279 in `lqcontrol.jl`, which - instead of returning a matrix as it should per https://github.com/QuantEcon/QuantEcon.jl/blob/5d25a26daf9a4e7858b7b29d8dfb505e7f0134fc/src/lqcontrol.jl#L311 - returns an `Array{Any,1}`.

What do you think? @spencerlyon2 Should I first change that in `lqcontrol.jl`, submit a separate PR, then fix the notebook?

Cheers,
Alberto

**Comment by spencerlyon2** [comment]
Hey @albep it seems like we need to fix lqcontrol.jl before we can move forward here so let's do it.

Thanks

**Comment by albep** [comment]
I'm going to do that asap

**Comment by albep** [comment]
the updated and working `lqcontrol_solutions_jl.ipynb` notebook is ready. When [this PR](https://github.com/QuantEcon/QuantEcon.jl/pull/130) is merged I'll make a PR here for it. Plots have been updated to Plots.jl with plotlyjs backend

**Comment by spencerlyon2** [comment]
That PR has been merged

**Comment by albep** [comment]
This can be closed :tada:

**Comment by spencerlyon2** [comment]
Great work
